src/js/utils/JSONUtil.js

```diff
@@ -0,0 +1,333 @@
+/**
```

Comment:
Yes, that particular one was pasted from another project on mine (and I think on it's turn it was initially copied somewhere from the interwebs). Will fix :+1: